backend/src/main/java/com/alessandraipiranga/backend/service/PlayerService.java

```diff
@@ -0,0 +1,48 @@
+package com.alessandraipiranga.backend.service;
+
+import com.alessandraipiranga.backend.model.PlayerEntity;
+import com.alessandraipiranga.backend.repo.PlayerRepository;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.EntityExistsException;
+import java.util.Optional;
+
+import static org.springframework.util.StringUtils.hasText;
+
+@Service
+@Getter
+@Setter
```

Comment:
`playerRepository` is final and you are using constructor injection (which is preferred). No need for getter and setter
```suggestion
```